docs/sources/configuration/_index.md

```diff
@@ -2368,7 +2368,7 @@ This way, one doesn't have to replicate configuration in multiple places.
 # For instance, you can expect the `heartbeat_period` defined in the common section
 # to be used by the distributor's ring, but only if the distributor's ring itself
 # doesn't have a `heartbeat_period` set.
-[ring: <ring_config>]
+[ring: <ring>]
```

Comment:
The same file `configuration/_index.md` have `<ring_config>` in other places as well (under `scheduler_ring` and `compactor_ring`). I think we should change the spelling there as well for the consistency?

Reply:
I'll update this @kavirajk.  Just checking: the scheduler_ring and compactor_ring references are supposed to go to the _common_ section's description of a ring?  Their ring configuration options are exactly the same as in the common section one? And, the only option is to specify ring details (for these 2 components) within the common section?

Reply:
Good point. May be @trevorwhitney or @DylanGuedes are the right people here. They have more context.

But we can take that in different PR anyway :)

Reply:
Yes, they are both seen as `pkg/util/ring_config.go` which is the same type used for the common ring. No, you don't have to specify their details within the common section; you have the option to define such things within their own configuration section (ex: `compactor: compactor_ring:` for the compactor).